src/App.tsx

```diff
@@ -20,12 +33,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={event => setQuery(event.target.value)}
```

Comment:
It's better to create hendler for that

Reply:
agree, create handler

Reply:
we use it once, why do we need to create separate handler?